go.mod

```diff
@@ -6,40 +6,31 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/creack/pty v1.1.7
-	github.com/digitalocean/godo v1.22.0
+	github.com/digitalocean/godo v1.25.1-0.20191114163902-6c8415f2abff
```

Comment:
Is this correct? This is pinned to a specific commit, could we just bump this to godo v1.26.0?